index.d.ts

```diff
@@ -20,10 +20,22 @@ export type Options = {
 	*/
 	readonly devToolsMode?:
 	| 'undocked'
+	| 'left'
 	| 'right'
 	| 'bottom'
 	| 'previous'
 	| 'detach';
+
+	/**
+	Allows to select windows to apply the debug options to.
+	Accepts a function to filter each window and returns:
+	- `true` to apply the given options
+	- `false` to skip the window (not apply debug)
+	- `Options` to override the global options
```

Comment:
readme and index.d.ts text should be in sync.

Reply:
I've added an extra commit improving the docs.
Tried to sync as much as possible and have the same format, but I'm sorry I couldn't see any real standards, nor there's anything that generates docs from code.

Please check and suggest any direct change if you want (also the PR allows edits by maintainers)